test/tls.uts

```diff
@@ -984,10 +1064,32 @@ t.len = 0xefff
 str(t) == b'\xde\xad\xbe\xef\xff\x01'
 
 
+= Building packets - TLS record with bad data
+a = TLS(b'\x00\x03\x03\x00\x03data')
+assert a.haslayer(Raw)
+
+
+= Building packets - _CipherSuitesField with no cipher
+from scapy.layers.tls.handshake import _CipherSuitesField
+a = _CipherSuitesField("test", None, {})
+assert a.i2repr(None, None) == "None"
+assert isinstance(a.randval(), RandBin)
+assert hash(a) == 8890308
```

Comment:
OK, I couldn't seem to make it work even with a seed, so I removed it altogether.